libs/wingsdk/.projenrc.ts

```diff
@@ -98,6 +98,7 @@ const project = new cdk.JsiiProject({
     "tar",
     "express",
     "uuid",
+    "nanoid@^3.3.6",
```

Comment:
You mentioned there is an issue with later versions of nanoid, maybe put a comment here to explain for future reference